src/components/EmailNotifications/EmailNotificationsForm.js

```diff
@@ -0,0 +1,295 @@
+          termsAccepted={termsAccepted}
+          setTermsAccepted={setTermsAccepted}
+        />
+
+        {isModal && (
+          <div
+            css={`
+              display: flex;
+              justify-content: space-between;
+              flex-direction: ${compactMode ? 'column' : 'row'};
+              width: 100%;
+              margin-bottom: ${1.5 * GU}px;
+              margin-top: ${3 * GU}px;
+            `}
+          >
+            <ActionButtons compactMode={compactMode} onClick={onOptOut}>
+              Opt out
+            </ActionButtons>
+            <ActionButtons
+              compactMode={compactMode}
+              mode="strong"
+              disabled={emailInvalid || !termsAccepted}
+              onClick={handleOnSubscribeToNotifications}
+            >
+              Subscribe
+            </ActionButtons>
+          </div>
+        )}
+      </div>
+    </>
+  )
+}
+
+function TextContent({ update, account, theme }) {
+  const content = update
+    ? `Enter a new email address for your account ${account}. We will continue sending email notifications to the current email address until you verify this new email address.`
+    : `Associate an email address to your account ${account}, so you can get notifications from all Aragon Court events.`
+
+  return (
+    <span
+      css={`
+        ${textStyle('body2')};
+        color: ${theme.surfaceContentSecondary};
+        margin-top: ${1.5 * GU}px;
+      `}
+    >
+      {transformAddresses(content, (part, isAddress, index) =>
+        isAddress ? (
+          <span title={part} key={index}>
+            <LocalIdentityBadge
+              connectedAccount={addressesEqual(part, account)}
+              entity={part}
+              compact
+            />
+          </span>
+        ) : (
+          <span key={index}>{part}</span>
+        )
+      )}
+    </span>
+  )
+}
+
+function LegalTermsAndPolicy({ termsAccepted, setTermsAccepted }) {
+  return (
+    <div
+      css={`
+        margin-top: ${3 * GU}px;
+      `}
+    >
+      <div
+        css={`
+          display: flex;
+        `}
+      >
+        <Checkbox
+          checked={termsAccepted}
+          onChange={checked => setTermsAccepted(checked)}
+        />
+
+        <span
+          css={`
+            ${textStyle('body2')};
+            color: #9096b6;
+            margin-left: ${1.5 * GU}px;
+            text-align: left;
+          `}
+        >
+          By continuing with your email, you agree to Aragon Court{' '}
+          <Link href="https://anj.aragon.org/legal/terms-general.pdf">
+            legal terms{' '}
+          </Link>{' '}
+          and{' '}
+          <Link href="https://aragon.one/email-collection.md">
+            {' '}
+            email collection policy.
+          </Link>
+        </span>
+      </div>
+    </div>
+  )
+}
+
+const ActionButtons = styled(Button)`
+  width: ${({ compactMode }) =>
+    compactMode ? '100% ' : `calc((100% - ${2 * GU}px) /  2)`};
+`
```

Comment:
Nitpick: we could use a react component for consistency (and to remove the styled-components import):

```suggestion
function ActionButtons({ compactMode }) {
  return (
    <Button
      css={`
        width: ${compactMode ? '100%' : `calc((100% - ${2 * GU}px) /  2)`};
      `}
    />
  )
}
```